### PR TITLE
fix(listItem): allow for multiple md-secondary items

### PR DESCRIPTION
**Pull request by OzzieOrca**
Addresses #2595
This allows the following code to produce the list items in the picture below:
```HTML
<md-list-item ng-repeat="person in people" ng-click="goToPerson(person)">
    <img class="md-avatar" ng-src="{{ person.picture }}">
    <p>{{ person.name }}</p>
    <md-checkbox class="md-secondary" ng-model="person.selected"></md-checkbox>
    <ng-md-icon icon="email" ng-click="sendEmail(person)" aria-label="Send Email" class="md-secondary"></ng-md-icon>
    <ng-md-icon icon="message" ng-click="sendText(person)" aria-label="Send Text" class="md-secondary"></ng-md-icon>
</md-list-item>
```
screen shot 2015-11-27 at 12 35 34 pm

Does this look like a decent solution? There might be a more general way to handle it or a better way. Let me know if you see any issues with it.

### CODE REVIEW

File: src/components/list/list.js [hunk 1]
```diff
@@ -5,11 +5,11 @@
...
-  'material.core'
-])
-  .controller('MdListController', MdListController)
-  .directive('mdList', mdListDirective)
-  .directive('mdListItem', mdListItemDirective);
+      'material.core'
+    ])
+    .controller('MdListController', MdListController)
+    .directive('mdList', mdListDirective)
+    .directive('mdListItem', mdListItemDirective);
```
Comment (DevVersion) [inline src/components/list/list.js]: That's not a good thing, to reformat this snippet.

File: src/components/list/list.js [hunk 2]
```diff
@@ -143,24 +143,41 @@ function mdListItemDirective($mdAria, $mdConstant, $mdUtil, $timeout) {
         tEl[0].setAttribute('tabindex', '-1');
         tEl.append(container);
 
+        if(secondaryItems.length === 1) {
```
Comment (DevVersion) [inline src/components/list/list.js]: Please space the bracket with the `if`

File: src/components/list/list.js [hunk 3]
```diff
@@ -202,18 +219,18 @@ function mdListItemDirective($mdAria, $mdConstant, $mdUtil, $timeout) {
...
+                    $scope.mouseActive = false;
+                  }, 100);
+                })
+                .on('focus', function() {
+                  if ($scope.mouseActive === false) { $element.addClass('md-focused'); }
+                  proxy.on('blur', function proxyOnBlur() {
+                    $element.removeClass('md-focused');
+                    proxy.off('blur', proxyOnBlur);
+                  });
+                });
```
Comment (DevVersion) [inline src/components/list/list.js]: On Angular Material we have strict rules with indention, please don't reformat existing code.
Indent with 2 Spaces

**Comment by OzzieOrca** [comment]
I think I've fixed those formatting issues. I used the auto indenting feature in my IDE and didn't check whitespace before committing. Let me know if you see any more.

**Comment by DevVersion** [comment]
Looks good now!. Thanks for fixing it :+1:

**Comment by gpopovic** [comment]
This is awesome.. Good work

**Comment by 8cells** [comment]
Could someone merge it?

**Comment by asm0dey** [comment]
:+1: 
Please, merge!

**Comment by justechn** [comment]
I saw that you tagged 1.0.5, but this was not included. Do you have any idea when this will be released?

**Comment by timmyrosen** [comment]
Also curious about this

**Comment by DevVersion** [comment]
@justechn @timmyrosen There is a new build process. Read more [here](https://github.com/angular/material#-our-release-processes)

**Comment by justechn** [comment]
@DevVersion I assume you are meaning that this will be in a minor release, not in a patch release? So it will be released sometime in the next month or two?

The only reason I asked is because @ThomasBurleson set the milestone as 1.0.5 7 days ago, but when 1.0.5 was release this was not included. If this is going to be in a minor release the milestone should be set to 1.1.0 or something similar.

**Pull request closed without merging**